### PR TITLE
Updates NPM packages

### PR DESCRIPTION
**Pull request by johnclary**

### CODE REVIEW

**Comment by johnclary** [comment]
👋 not asking for y'all to test: i was thinking i would merge tomorrow's release candidate into this branch and run the complete Moped release tests against it.

so just an FYI!

File: moped-editor/package.json [hunk 1]
```diff
@@ -100,7 +100,8 @@
     "npm": ">=6.0.0"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```
Comment (johnclary) [inline moped-editor/package.json:103]: this fixes this problem:
https://stackoverflow.com/questions/76435306/babel-preset-react-app-is-importing-the-babel-plugin-proposal-private-propert

**Review by chiaberry — APPROVED** [review]
✨

**Review by mddilley — APPROVED** [review]
🚢 🙏

**Pull request merged**